.github/workflows/tests.yml

```diff
@@ -85,6 +86,14 @@ jobs:
               core.setOutput('force-linux', 'false')
             }
 
+            if (label_names.includes('CI-linux-wheezy')) {
+              console.log('CI-linux-wheezy label found. Running Linux wheezy builds.')
+              core.setOutput('linux-wheezy', 'true')
```

Comment:
We can output the image name here, and the just use the output in `docker run`.
Once those 3 env vars below are gone ofc.